python/tests/io/test_hdf5.py

```diff
@@ -4,8 +4,8 @@
 # @author Simon Heybrock
 import scipp as sc
 import numpy as np
-
 import tempfile
+import pytest
 
 
 def check_roundtrip(obj):
```

Comment:
Minor point. All tests write the same file name. It may be useful for debugging purposes if file names reflected the test name.

Reply:
I don't think they write the same filename. I am using `tempfile` to create a temporary directory for every test. The naming within that dir is the same. The context manager is removing the directory immediately.